src/hb-ot-post-table.hh

```diff
@@ -170,6 +170,8 @@ struct post
       hb_bytes_t st (name, len);
       const uint16_t *gid = (const uint16_t *) hb_bsearch (hb_addressof (st), gids, count,
 							   sizeof (gids[0]), cmp_key, (void *) this);
+      free(gids);
```

Comment:
This is intentional as the allocated array is actually stored in `gids_sorted_by_name.cmpexch (nullptr, gids)` line for future uses, AFAIK

Reply:
We see use `gids` allocated in the `hb_bsearch()` function, CC: @behdad

Reply:
What @ebraminio said.  This is freed when the accelerator is fini()ed eventually.